src/mumble/MainWindow.cpp

```diff
@@ -1380,6 +1388,35 @@ void MainWindow::setOnTop(bool top) {
 	}
 }
 
+void MainWindow::applyState(const bool minimalView) {
+	if (minimalView) {
+		if (!Global::get().s.qbaMinimalViewGeometry.isNull()) {
+			restoreGeometry(Global::get().s.qbaMinimalViewGeometry);
```

Comment:
This should also use the UI version number, no?

Reply:
The function doesn't take an argument for that.

Reply:
Oh - right this is only about the geometry, nothing else 💡